sympy/vector/coordsysrect.py

```diff
@@ -346,19 +317,16 @@ def _connect_to_standard_cartesian(cls, curv_coord_type, inverse=True):
         Parameters
         ==========
 
-        param curv_coord_type: str, tuple
+        curv_coord_type: str, tuple
 
-        variables : Tuple
-            The variables which equations depends on.
+        variables : tuple
+            Tuple of variables, which transformation equations depends on.
 
         """
 
         x1, x2, x3 = symbols("x1, x2, x3", cls=Dummy)
 
         if isinstance(curv_coord_type, string_types):
-            if variables is None:
-                variables = _default_variables(curv_coord_type)
-
             transformation_equations = _set_transformation_equations_mapping(curv_coord_type)
```

Comment:
`transformation_equations` is a `lambda`, we need a SymPy `Lambda` (capital L) for hashing and internal stuff of SymPy.